Makefile

```diff
@@ -38,3 +51,8 @@ _data/versions.json: node_modules docs
 node_modules: package.json package-lock.json
 	npm install
 	@touch node_modules
+
+# Create the local docs dir needed before _data/versions.json in some cases
+local_docs_dir: 
```

Comment:
If this is making the directory, I would recommend that we use `$(LOCAL_DOCS_DIR)` as the name of this recipe. Then we get dependency checking and incremental building for free!